NativeAuthSampleApp/ProtectedAPIViewController.swift

```diff
@@ -87,23 +92,45 @@ class ProtectedAPIViewController: UIViewController {
         accountResult?.signOut()
 
         accountResult = nil
-        
-        accessToken = nil
+        accessTokenAPI1 = nil
+        accessTokenAPI2 = nil
 
         showResultText("Signed out")
-
         updateUI()
     }
     
-    @IBAction func protectedAPIPressed(_: Any) {
-        guard let accessToken = self.accessToken else {
-            let errorMessage = "No access token found"
-            print(errorMessage)
-            showResultText(errorMessage)
+    @IBAction func protectedApi1Pressed(_: Any) {
+        guard   let url = protectedAPIUrl1,
+                !protectedAPIScopes1.isEmpty else {
+            showResultText("API 1 not configured.")
+            return
+        }
+        
+        if let accessToken = accessTokenAPI1 {
+            accessProtectedAPI(apiUrl: url, accessToken: accessToken)
+        } else {
+            accountResult?.getAccessToken(scopes: scopesApi1, delegate: self)
+            let message = "Retrieving access token to use with API 1..."
+            showResultText(message)
+            print(message)
+        }
+    }
+    
+    @IBAction func protectedApi2Pressed(_: Any) {
+        guard   let url = protectedAPIUrl2,
+                !protectedAPIScopes2.isEmpty else {
+            showResultText("API 2 not configured.")
             return
         }
         
-        accessProtectedAPI(accessToken: accessToken)
+        if let accessToken = accessTokenAPI2 {
+            accessProtectedAPI(apiUrl: url, accessToken: accessToken)
+        } else {
+            accountResult?.getAccessToken(scopes: scopesApi2, delegate: self)
```

Comment:
This should be "protectedAPIScopes2"
```suggestion
            accountResult?.getAccessToken(scopes: protectedAPIScopes2, delegate: self)
```

Reply:
Fixed